### ISSUE TITLE
Add docs for build

### ISSUE THREAD

**Issue by TimTheBig** [jyjblrd/4-Axis-Polar-3D-Printer#3]
I'm interested in a build to add support to my own slicer however the lack of instructions is a big turn off

**Comment by xobes** [comment]
What is is you're looking for?  A step-by-step for the physical build? Or software "build" instructions?

**Comment by TimTheBig** [comment]
The physical build would be great.

**Comment by greywolf4002** [comment]
Yeah I agree, I might start this project sometime at the end of January so if nothing is updated I'll make a list on my own.

**Comment by jyjblrd** [comment]
Hi, unfortunately I don't have the time to make a build guide but please feel free to make one and upload it as a PR, it would really help the community

**Comment by xobes** [comment]
I will be doing this in the coming weeks, am doing a build presently and taking lots of pics

**Comment by xobes** [comment]
I have a lot of pictures, but I've also discovered I was missing some parts, so while I wait for those, I've made some modifications to several b-axis parts and so my "build instructions" and screw counts aren't going to be 100% aligned with the original -- partly due to the fact that I bought button-head screws instead of both flat-head and socket-cap screws.

I'll be starting the documentation in my fork, and will not deviations accordingly.  I do think that these things will improve as more people go through the motions such that more optimizations can be made as things progress.

**Comment by pele1659** [comment]
hello i can't find an 3D model for bilding the printer.
the only model i fond is https://mycam641.autodesk360.com/g/shares/SH30dd5QT870c25f12fcf82c385dbe23a74f that.
where can i find the model for the bilding?

**Comment by jyjblrd** [comment]
it should be called "extruder mount"

**Comment by pele1659** [comment]
Thank you for the reply. 
I downloaded again the zip file with all the files, but I still see only one part. 
I tray to open it in another computer and get the same result.

**Comment by oldnslow** [comment]
@pele1659 
> I downloaded again the zip file with all the files, but I still see only one part.

Please forgive me if i misunderstand your issue, but just to clarify:
Are you looking for the .stl files required to print the parts or are you looking for instructions on how to assemble the printer?

If you are looking for the .stl files you should find them on the git site here:
![Image](https://github.com/user-attachments/assets/31ace5c1-66bc-4980-a95d-99151994bd2a)

After you **_extract_** the files from the .zip folder you should find the .stl files here:

![Image](https://github.com/user-attachments/assets/6ffb4cd6-a8f6-4336-9a18-fb7c22c8c757)

**Comment by oldnslow** [comment]
Ah, very good, you are a little ways ahead of me!  My understanding is that there are no detailed assembly instructions at this time - one of us Core-R-Theta enthusiasts will have to fill that breach!

You can  however take a look at the printer components as an assembly to see what goes where via the 4 axis polar 3d printer.step file which i was able to open in [FreeCad](https://www.freecad.org/).
![Image](https://github.com/user-attachments/assets/f73fc061-c530-440e-b91e-8aee00aa8ce1)

or via the 4 axis polar 3d printer.f3z file which, **_if you first rename it to .zip and then extract the files_** (should find four .f3d files),
![Image](https://github.com/user-attachments/assets/6690f3e2-5b46-4990-9604-2ce83b00ccca)

you can then open with the the [Autodesk Viewer](https://viewer.autodesk.com/designviews) (requires an account and a little patience, i had to move the extracted files out of the original .zip directory to get it to work).
![Image](https://github.com/user-attachments/assets/71593bc7-5d54-4a77-8732-377162b202de)
